src/LogFileReader.cpp

```diff
@@ -48,11 +53,35 @@ bool LogFileReader::readAll(const QString& fileName)
 bool LogFileReader::parseLine(const QString& line, BatteryData& batteryData) const
 {
     // TODO implement this first
+    QStringList splitLine = line.split(", ");
```

Comment:
Also, ", " is called a Delimiter (Characters that split up a text file) and should actually initializd in the namespace as a constant String.